kvbc/src/resources-manager/AdaptivePruningManager.cpp

```diff
@@ -26,10 +32,18 @@ void AdaptivePruningManager::notifyReplicas(const long double &rate, const uint6
   concord::messages::PruneTicksChangeRequest pruneRequest;
 
   pruneRequest.sender_id = bftClient->getClientId();
-  pruneRequest.tick_period_seconds = 1;
+  pruneRequest.ticks_per_second = 1;
 
   pruneRequest.batch_blocks_num = rate / bftEngine::ReplicaConfig::instance().numReplicas;
 
+  // Is this going to register all send values or just update current
+  ticksPerSecondMetric.Get().Set(pruneRequest.ticks_per_second);
+  batchSizeMetric.Get().Set(pruneRequest.batch_blocks_num);
```

Comment:
we also need to send metrics from the resource manager e.g. post-execution utilization, pruning utilization, pruning avg time etc

Reply:
For simplicity, the pruning info struct can be extended to include those values and reported here